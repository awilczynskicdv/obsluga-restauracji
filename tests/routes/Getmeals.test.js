import { callApi } from "../helpers.js";

const PATH = `api/private/meals`;

test(`[GET] ${PATH} - valid request - should return status 200`, async () => {
  const response = await callApi("get", "/meals/all", {}, {}, false);
 expect(response.data).toMatchSnapshot();
 expect(response.status).toEqual(200);
});
