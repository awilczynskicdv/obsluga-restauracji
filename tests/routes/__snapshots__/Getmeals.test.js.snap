// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[GET] api/private/meals - valid request - should return status 200 1`] = `"Not enough meals in menu"`;
